src/main/kotlin/uk/gov/justice/digital/hmpps/suicideriskformapi/exception/NotFoundException.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.breachnoticeapi.exception
```

Comment:
Need to update this package name
```suggestion
package uk.gov.justice.digital.hmpps.suicideriskformapi.exception
```